frontend/src/pages/article/article-page.tsx

```diff
@@ -81,13 +81,14 @@ const ArticlePage: React.FC = () => {
   }, [location]);
 
   useEffect(() => {
-    void dispatch(articleActions.getArticle(Number(id)));
+    void dispatch(articleActions.getArticle(Number(id))).then(scrollToComments);
```

Comment:
Won't it behave the way that on article page opening there would always be scrolling to comments?

Reply:
No, it won't. It scrolls only if we click on the comments icon, because only in this case `hash` has a value